include/dgl/immutable_graph.h

```diff
@@ -1000,6 +1009,9 @@ class ImmutableGraph: public GraphInterface {
   std::string shared_mem_name_;
   // We serialize the metadata of the graph index here for shared memory.
   NDArray serialized_shared_meta_;
+
+  mutable bool is_unibipartite_set_ = false;
+  mutable bool is_unibipartite_ = false;
```

Comment:
Better add a description for both flags like the other private members.

Reply:
Added descriptions in 11024ec.